### PR TITLE
Make Messlokation➡Netzebene Messung nilable (potentially breaking)

### PR DESCRIPTION
**Pull request by hf-kklein**
This is potentially breaking

### CODE REVIEW

**Review by qndaa — APPROVED** [review]

File: bo/messlokation_test.go [hunk 1]
```diff
@@ -32,7 +33,7 @@ func (s *Suite) Test_Messlokation_Deserialization() {
 		},
 		MesslokationsId:              "DE0000011111222223333344444555556",
 		Sparte:                       sparte.STROM,
-		NetzebeneMessung:             netzebene.MD,
+		NetzebeneMessung:             internal.Ptr(netzebene.MD),
```
Comment (hf-krechan) [inline bo/messlokation_test.go:36]: what does the `internal.Ptr()`?

**Review by hf-krechan — APPROVED** [review]
lgtm

File: bo/messlokation_test.go (hunk 1 above)
Comment (hf-kklein) [reply bo/messlokation_test.go:36]: internal.Ptr(netzebene.MD) = &netzebene.MD

aber in kompliziert, weil go so einfach ist.

Comment (hf-krechan) [reply bo/messlokation_test.go:36]: I see, trivial knowledge

**Pull request merged**